src/styles/content.scss

```diff
@@ -275,4 +275,9 @@ table.os-raise-horizontaltable {
 
 .os-raise-text-bold {
   font-weight: 700;
+}
+
+.os-raise-img-fluid {
+  max-width: calc(100% - 1rem);
```

Comment:
Just out of curiosity: This seems pretty subtle to me from a quick test. Is there a specific scenario where it makes a bigger difference, or just a good practice to create a bit of space?

Reply:
I wouldn't say a big difference, but one particular example where it might be useful is shown below. I actually was going back and forth between just making it `100%`. Now that I've had time to sit on it + you noting it, I think that may be better and any spacing between content should be handled differently. I originally was also trying to mirror what was used inside of Moodle to try and keep some consistency.

Notice because of the `1 rem` the image doesn't touch the table before it wraps to the top.
https://github.com/openstax/k12-apps-raise/assets/37000585/dd95dbea-48cc-4396-b931-7034c882a6a9

Reply:
It's probably fine either way, but I'll defer to your judgment on what might be most consistent.